### PR TITLE
Issue #11 - Invalid key type

### PR DESCRIPTION
**Pull request by Bakobiibizo**
Added check for key2address to stop error from being thrown from wrong key type
Comx key balances would throw an error because key2address was in the incorrect format for it to check the values of. I added a conditional check for the name=="key2address" to handle this edge case. 
A more robust solution will need to be found if you want to actually check the balance of key2address rather than just omit it from the balances.

### CODE REVIEW

**Comment by steinerkelvin** [comment]
Why would be a file called `key2address` in the keys folder?
Can you send me a sample?

**Comment by Bakobiibizo** [comment]
commune just makes it. no clue why. 

```{"data": {"key2address": "5CkHmfkxN1dHuaddt9r5Ur9hxC2Ex5mJ4Qm46f9VMfXWmZ9L"}, "encrypted": false, "timestamp": 1709791965}```

**Comment by Bakobiibizo** [comment]
image

**Comment by steinerkelvin** [comment]
> commune just makes it. no clue why.

the bane of my existence.


Yeah, it's fine to ignore it. Just print a warning, so the user is informed something is being ignored and is not caught by surprise.

(I hope no one is naming a key `key2address`, but we should get used to writing our code defensively like this anyway)

File: src/communex/compat/key.py [hunk 1]
```diff
@@ -142,6 +142,9 @@ def local_key_addresses() -> dict[str, Ss58Address]:
     addresses_map: dict[str, Ss58Address] = {}
 
     for key_name in key_names:
+        # issue #11 https://github.com/agicommies/communex/issues/12 added check for key2address to stop error from being thrown by wrong key type. 
+        if key_name == "key2address": 
+            continue
```
Comment (steinerkelvin) [inline src/communex/compat/key.py:147]: Print a warning plz

**Review by steinerkelvin — CHANGES_REQUESTED** [review]

File: src/communex/compat/key.py (hunk 1 above)
Comment (Bakobiibizo) [reply src/communex/compat/key.py:148]: updated

**Pull request merged**